index.html

```diff
@@ -148,9 +144,8 @@ <h2 class="project-title">Mult-Post Stories</h2>
                         <li>javaScript</li>
                     </ul>
                     <button class="button" type="button">See Project</button>
-                </article>
-            </div>
-        </section>
+            </section>
+        </div>
 
         <!-- about myself -->
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.